contracts/SSVNetwork.sol

```diff
@@ -807,6 +756,33 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
         emit OperatorFeeExecuted(msg.sender, operatorId, block.number, fee);
     }
 
+    function _updateOperators(
+        uint64[] memory operatorIds,
+        bool increaseValidatorCount,
+        uint32 deltaValidatorCount
+    ) internal returns (uint64 clusterIndex, uint64 burnRate) {
+        uint operatorsLength = operatorIds.length;
+
+        for (uint i; i < operatorsLength; ) {
+            Operator memory operator = operators[operatorIds[i]];
+            if (operator.snapshot.block != 0) {
+                operator.updateSnapshot();
+                if (increaseValidatorCount) {
+                    operator.validatorCount += deltaValidatorCount;
+                } else {
+                    operator.validatorCount -= deltaValidatorCount;
+                }
+                burnRate += operator.fee;
+                operators[operatorIds[i]] = operator;
+            }
+
+            clusterIndex += operator.snapshot.index;
+            unchecked {
+                ++i;
+            }
+        }
+    }
```

Comment:
shouldnt we have an additional line after the for loop with something like:

burnRate += networkFee * validatorCount

Reply:
Good observation @andrew-blox , thanks. after calling `_updateOperators` , there is a call to either `ClusterLib.updateBalance` or `ClusterLib.updateClusterData` where the `networkFee` and `validatorCount` is taking into account to update the cluster's balance.